### PR TITLE
Proposed changes for linking and tabindexing

### PR DESCRIPTION
**Pull request by bkardell**
Feel free to reject this, just trying to highlight what I think we need and where for discussion re https://github.com/mathml-refresh/mathml/issues/125 and to unblock answers for https://github.com/mathml-refresh/mathml/issues/152...

### CODE REVIEW

File: webidl/MathMLLinkableElement.idl [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+interface MathMLLinkableElement : MathMLElement { 
+  [CEReactions] attribute DOMString target;
+  [CEReactions] attribute DOMString download;
+  [CEReactions] attribute USVString ping;
+  [CEReactions] attribute DOMString rel;
+  [SameObject, PutForwards=value] readonly attribute DOMTokenList relList;
+  [CEReactions] attribute DOMString hreflang;
+  [CEReactions] attribute DOMString type;
+
+  [CEReactions] attribute DOMString text;
```
Comment (rwlbuis) [inline webidl/MathMLLinkableElement.idl]: what is this? Alt text?

File: webidl/MathMLLinkableElement.idl [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[Exposed=Window]
+interface MathMLLinkableElement : MathMLElement { 
+  [CEReactions] attribute DOMString target;
```
Comment (rwlbuis) [inline webidl/MathMLLinkableElement.idl]: It seems like this should closely follow:
https://html.spec.whatwg.org/#the-link-element.
So then href may be better?

File: webidl/MathMLLinkableElement.idl [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  [CEReactions] attribute DOMString download;
```
Comment (rwlbuis) [inline webidl/MathMLLinkableElement.idl]: Where does this come from?

Comment (rwlbuis) [reply webidl/MathMLLinkableElement.idl]: Ah, you are following https://html.spec.whatwg.org/#the-a-element.

File: webidl/MathMLLinkableElement.idl (hunk 1 above)
Comment (bkardell) [reply webidl/MathMLLinkableElement.idl]: from `a` in HTML.. 

> a . text
Same as textContent.

File: webidl/MathMLLinkableElement.idl (hunk 2 above)
Comment (bkardell) [reply webidl/MathMLLinkableElement.idl]: based on https://html.spec.whatwg.org/#the-a-element, not link

**Comment by bkardell** [comment]
> Only thing you may want to add is a constructor.

Since there is no concrete element here, I think we don't want to add that.  It is, effectively, an abstract base class, basically?

**Pull request merged**